superset/databases/schemas.py

```diff
@@ -1116,12 +1116,6 @@ class CSVUploadPostSchema(BaseUploadPostSchema):
             "description": "DD/MM format dates, international and European format"
         }
     )
-    overwrite_duplicates = fields.Boolean(
-        metadata={
-            "description": "If duplicate columns are not overridden,"
-            "they will be presented as 'X.1, X.2 ...X.x'."
-        }
-    )
```

Comment:
@dpgaspar will this break an existing api if someone is passing this field? Should we deprecate it instead?

Reply:
this was only used on the new API introduced here: https://github.com/apache/superset/pull/27840
so this API was not yet released